### PR TITLE
Add a Thorough Guide for Beginners

### PR DESCRIPTION
**Pull request by lsafd**
This new file contains a guide that beginners can follow to install Heads without any prior experience.

After struggling to install Heads, I noticed that the existing installation documentation is scattered. Some is contained in Building.md, Installing-Heads.md, Clean-the-ME-firmware.md and in other files. The files you need to reference if you are installing for the first time are not ordered and you have to move back and forth.

While it is good that the detailed documentation is there for those who can understand it, I think that there should be at least a succinct first installation guide for new users. The guide I have proposed is not complete and I plan to add pictures that further detail the steps. However, I think that even in its current state it would be helpful to the project.

### CODE REVIEW

File: Beginner-Installation-Guide.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+make BOARD=x230
+```
+
+Make Heads for another board (`XXX` should be the name of your board in ./boards):
+
+```
+make BOARD=XXX
+```
+
+The resulting rom file can be found in `./build/XXX/XXX.rom` (`XXX` should be the name of your board in `./boards`).
```
Comment (tlaurion) [inline Beginner-Installation-Guide.md:128]: Or coreboot.rom (x230 outcome)

File: Beginner-Installation-Guide.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+Try to read the name on the top SPI flash chip. Then, connect the clip and ch341a programmer to the top SPI flash chip. Use flashrom to check the chip that you are connected to:
+
+```
+sudo flashrom -p ch341a_spi
+```
+
+Find the chip and read from it twice (For me the SPI flash chip is `YYY`):
+
+```
```
Comment (tlaurion) [inline Beginner-Installation-Guide.md:153]: I would recommend doing -v instead to verify rom image against of rereading and diffing

**Comment by tlaurion** [comment]
Awsesome! Some minor corrections and suggestions!

**Comment by tlaurion** [comment]
@lsafd apart from minor change requests, I love it!

**Comment by lsafd** [comment]
I changed the things that you suggested. Let me know what you think!

**Pull request merged**